lib/collection.rb

```diff
@@ -0,0 +1,26 @@
+require 'csv'
+
+class Collection
+  attr_reader :collection
+
+  def initialize
+    @collection = []
+  end
+
+  def self.from_csv(csv_location)
+    new_collection = self.new
+    new_collection.load_csv(csv_location)
+    new_collection.collection
+  end
+
```

Comment:
Every Collection class now has a class method -- it will be invoked by the StatTracker .from_csv